genie-core/src/main/java/com/netflix/genie/core/services/impl/RandomizedClusterLoadBalancerImpl.java

```diff
@@ -52,4 +54,12 @@ public Cluster selectCluster(final List<Cluster> clusters) throws GenieException
         final Random rand = new Random();
         return clusters.get(Math.abs(rand.nextInt(clusters.size())));
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public int getOrder() {
+        return Ordered.LOWEST_PRECEDENCE;
```

Comment:
We want this to always go last. What happens if another LB implementation also uses LOWEST_PRECEDENCE?
PriorityOrdered only works the other way around (giving higher precedence if two objects have the same precedence value).